packages/devtools_app/lib/src/flutter/controllers.dart

```diff
@@ -67,7 +67,7 @@ class Controllers extends StatefulWidget {
 
   static ProvidedControllers of(BuildContext context) {
     final _InheritedProvider inherited =
-        context.inheritFromWidgetOfExactType(_InheritedProvider);
+        context.dependOnInheritedWidgetOfExactType<_InheritedProvider>();
```

Comment:
With this new method we can remove the left-hand _InheritedProvider type.  With the deprecated old method, we had to cast from InheritedWidget to _InheritedProvider

cc @albertusangga